tests/system/test_monitoring.py

```diff
@@ -1,43 +0,0 @@
-import os
-from urllib.parse import urlparse, urlunparse
-
-from elasticsearch import Elasticsearch
-from apmserver import integration_test
-from apmserver import ElasticTest
-from helper import wait_until
-
-
-@integration_test
-class Test(ElasticTest):
-    def config(self):
-        cfg = super(Test, self).config()
-
-        # Extract username/password from the URL, and store
-        # separately from elasticsearch_host. This is necessary
-        # because Beats will use the userinfo from the URL in
-        # preference to specified values.
-        url = urlparse(cfg["elasticsearch_host"])
-        if url.username:
-            cfg["elasticsearch_username"] = url.username
-        if url.password:
-            cfg["elasticsearch_password"] = url.password
-        bare_netloc = url.netloc.split('@')[-1]
-        url = list(url)
-        url[1] = bare_netloc
-
-        # Set a password for the built-in apm_system user, and use that for monitoring.
```

Comment:
@simitt pointed out to me that this is the key part of the test, and that what we're testing here is that the built-in `apm_system` user has permissions to create monitoring docs. I'll convert this to another Go test.